watch/watch.1

```diff
@@ -7,41 +7,54 @@
 .SH NAME
 watch \- Keep an eye on a command output.
 .SH SYNOPSIS
-\fBwatch\fR \fB\-n\fR [\fIseconds\fR] [\fB\-bt\fR] [\fBcommand\fR [\fIargs\fR...]]
+\fBwatch\fR [\fB\-n\fR \fIseconds\fR] [\fB\-bt\fR] \fBcommand\fR [\fIargs\fR...]
 .SH DESCRIPTION
 The
 .B watch 
-utility executes a command repeatedly every "user-defined" seconds.
-This allows you to watch a program output change over the time.
+utility executes a command repeatedly every
+"user-defined" seconds. This allows you to watch
+a program output change over the time.
 .PP
 It accepts the following options:
 .TP
 .B \-n
-Specifies the interval to refresh the output. Per default, the
-interval is of 2 seconds.
+Specifies the interval to refresh the output.
+Per default, the interval is of 2 seconds.
```

Comment:
```suggestion
By default, the interval is of 2 seconds.
```

Reply:
Romanesque influence, I beg your pardon. Fixing this too.